apps/o2-blocks/src/callout/block.json

```diff
@@ -0,0 +1,45 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "a8c/callout",
+	"version": "1.0.0",
+	"title": "Callout",
+	"description": "Draw attention with a styled callout box.",
+	"icon": "megaphone",
+	"category": "a8c",
+	"keywords": [ "callout", "info", "warning", "tip", "example" ],
+	"textdomain": "o2-blocks",
+	"supports": {
+		"align": [ "wide", "full" ],
+		"html": false
+	},
+	"attributes": {
+		"calloutTemplate": {
+			"type": "string",
+			"default": "custom"
+		},
+		"customColor": {
+			"type": "string",
+			"default": "#50575e"
```

Comment:
why are these custom attributes set with default values? how do we indicate that there’s no custom color?